lib/hammer_cli_foreman/compute_resource.rb

```diff
@@ -121,7 +121,7 @@ class AvailableNetworksCommand < HammerCLIForeman::ListCommand
       command_name 'networks'
 
       output do
-        field :id, _('Id')
+        field :id, _('Id'), Fields::List, :width => 200, :hide_blank => true
```

Comment:
For this field `Fields::Field` is enough and more readable in my opinion.

And if we want the field to not be static (200 chars only), but squeezed or expanded according to the data (e.g. 20 chars data => 20 chars width of the column), please change the `:width` option to `:max_width`.

Reply:
@ofedoren Made changes as per suggestions. Let me know if I missed something.